File: PurwokertoExplore/app/src/main/java/com/tubes/wisata/database/AppDatabase.java
package com.tubes.wisata.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.tubes.wisata.database.dao.WisataDao;
import com.tubes.wisata.model.ModelWisata;


@Database(entities = {ModelWisata.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract WisataDao wisataDao();
}

File: PurwokertoExplore/app/src/main/java/com/example/purwokertoexplore/database/AppDatabase.kt
package com.example.purwokertoexplore.database

import androidx.room.Database
import com.example.purwokertoexplore.model.ModelDatabase
import androidx.room.RoomDatabase
import com.example.purwokertoexplore.database.dao.DatabaseDao



@Database(entities = [ModelDatabase::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun databaseDao(): DatabaseDao?
}